test/stdin/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+cd "$(dirname "$0")"
+
+function fail {
+  msg=$1 ; shift
+  echo "FAIL $msg" >&2
+  for line in "$@"; do
+    echo "$line" >&2
+  done
+  exit 1
+}
+
+rm -rf node_modules
+mkdir -p node_modules
+ln -s ../../..                       node_modules/estrella
+ln -s ../../../node_modules/esbuild  node_modules/esbuild
+
+export ESTRELLA_TEST_VERBOSE=$ESTRELLA_TEST_VERBOSE
+if [ "$1" == "-verbose" ]; then
+  export ESTRELLA_TEST_VERBOSE=1
+fi
+
+./build.js -quiet <<< 'export function working() { return "works" }'
```

Comment:
nit: `<<<` is bash specific and relies on temporary files. A more portable form:

```sh
echo 'export function working() { return "works" }' | ./build.js -quiet
```

It's up to you which you prefer. Just my 2¢ :-)

Reply:
Since there is a bash shebang, I thought this read more like a sentence than the echo version. I'll change it for sure, but I think this reads more like "build (quietly) this function" in my head.

Reply:
If you think it reads better with `<<<` then let's keep it that way